src/util.rs

```diff
@@ -89,13 +67,18 @@ pub(crate) fn init_policy_engine(
         engine.enable_coverage();
     }
 
+    // Add the standard semconv policies
+    // Note: `add_policy` the package name, we ignore it here as we don't need it
+    _ = engine
+        .add_policy("defaults/rego/semconv.rego", SEMCONV_REGO)
```

Comment:
Should this only be added for a particular phase of the engine?  Aren't these symbols meaningless in `before_resolution` or are they meant to be applied there too?

Reply:
IMO, some of the rules/functions definitions could be used in any phases.